.gitignore

```diff
@@ -21,7 +21,11 @@ mock_*
 
 # Misc
 .DS_Store
+ignore/
```

Comment:
what's the rationale for ignoring this directory?

Reply:
that's a folder i use to put random stuff like patches, etc, so i don't have to see them in in my git view.

Reply:
unrelated to this PR but useful